### PR TITLE
feat(IR-2630): adding missing classes from designs, adding secondary …

### PR DESCRIPTION
**Pull request by kattsushi**
- feat(IR-2630): adding missing classes from desings, adding secondary button variant

## References
related Jira ticket https://tsu.atlassian.net/browse/IR-2630

### CODE REVIEW

**Review by SYBIOTE — APPROVED** [review]
Great, looks good

File: packages/client-core/src/common/services/ThemeService.tsx [hunk 1]
```diff
@@ -56,9 +63,14 @@ const lightTheme: CSSClasses = {
   'bg-primary': '#F5F5F5',
   'bg-secondary': '#FFFFFF',
   'bg-highlight': '#D9D9D9',
+  'bg-accent': '#375DAF',
```
Comment (HexaField) [inline packages/client-core/src/common/services/ThemeService.tsx]: There are so many duplicates here. Do we really need so many with the same colour?

**Review by HexaField — DISMISSED** [review]

File: packages/ui/src/primitives/tailwind/Button/index.tsx [hunk 2]
```diff
@@ -32,7 +32,16 @@ export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
   endIcon?: ReactNode
   children?: ReactNode
   size?: 'small' | 'medium' | 'large'
-  variant?: 'primary' | 'outline' | 'danger' | 'success' | 'pink' | 'transparent' | 'onboarding' | 'sidebar'
+  variant?:
```
Comment (HexaField) [inline packages/ui/src/primitives/tailwind/Button/index.tsx]: This is getting to a lot of variants, are they all actually used?

File: packages/client-core/src/common/services/ThemeService.tsx (hunk 1 above)
Comment (kattsushi) [reply packages/client-core/src/common/services/ThemeService.tsx]: good catch that i didn't realized, fixed removed the duplicates

File: packages/ui/src/primitives/tailwind/Button/index.tsx (hunk 2 above)
Comment (kattsushi) [reply packages/ui/src/primitives/tailwind/Button/index.tsx]: Checking all variants are being used except pink, so I just removed it

File: packages/ui/src/primitives/tailwind/Button/index.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
   endIcon?: ReactNode
   children?: ReactNode
   size?: 'small' | 'medium' | 'large'
-  variant?: 'primary' | 'outline' | 'danger' | 'success' | 'pink' | 'transparent' | 'onboarding' | 'sidebar'
+  variant?: 'primary' | 'secondary' | 'outline' | 'danger' | 'success' | 'transparent' | 'onboarding' | 'sidebar'
```
Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/Button/index.tsx]: we do not have onboarding variant anymore in the button

**Review by aditya-mitra — APPROVED** [review]
Looks good. Just need to resolve 1 comment

File: packages/client-core/src/common/services/ThemeService.tsx [hunk 4]
```diff
@@ -62,9 +67,13 @@ const lightTheme: CSSClasses = {
   'bg-primary': '#F5F5F5',
   'bg-secondary': '#FFFFFF',
   'bg-highlight': '#D9D9D9',
+  'bg-surface-bg': '#FFFFFF',
   'bg-surface-main': '#FFFFFF',
```
Comment (aditya-mitra) [inline packages/client-core/src/common/services/ThemeService.tsx:71]: can you also update surface main color to `#0E0F11`?

**Pull request merged**